source/documentation/other-topics/modsecurity.md

```diff
@@ -0,0 +1,108 @@
+
+ModSecurity is a normally third party add-on for nginx, however this is included within the [nginx ingress-controller](https://github.com/kubernetes/ingress-nginx) package for Kubernetes. 
+
+There are two ways to implement ModSecurity:
+  
+ - It can be enabled on a cluster level using ConfigMaps - this will enable ModSecurity for all paths, and each path must be disabled manually.
+ - It can be enabled on an Ingress level (individually) using `annotations`.
+
+For the Cloud Platform, ModSecurity has been configured as an opt-in feature. New and current applications will require a specific set of annotations to be added to their ingress manifest file.
+
+#### OWASP Rules
+
+The [OWASP ModSecurity Core Rule Set (CRS)](https://coreruleset.org/faq/) is a set of generic attack detection rules for use with ModSecurity. The CRS aims to protect web applications from a wide range of attacks, including the OWASP Top Ten, with a minimum of false alerts. The CRS provides protection against many common attack categories, including:
+
+ - SQL Injection (SQLi)
+ - Cross Site Scripting (XSS)
+ - Local File Inclusion (LFI)
+ - Remote File Inclusion (RFI)
+ - PHP Code Injection
+ - Shellshock
+ - Unix/Windows Shell Injection
+ - Session Fixation
+ - Scripting/Scanner/Bot Detection
+ - Metadata/Error Leakages
+
+##### Paranoia Level
+
+The Paranoia Level (PL) setting allows you to choose the desired level of rule checks. For the Cloud Platform implementation, this has been set to PL1. For more information on Paranoia Levels, please go to the `What are paranoia levels, and which level should I choose?` section [here](https://coreruleset.org/faq/)
+
+##### Anomaly Scoring Mode
+
+Traditional Detection or Passive Mode is the most basic operating mode where all of the rules are run as individual entities. In this mode no intelligence is shared between rules and each rule has no information about any previous rule matches. That is to say, in this mode, if a rule triggers, it will execute any disruptive/logging actions specified on the current rule.
+
+Anomaly scoring mode implements the concept of Collaborative Detection and Delayed Blocking. Rule logic has been set to decouple the inspection/detection from the blocking functionality. The individual rules can be run so that the detection remains, however instead of applying any disruptive action at that point, the rules will contribute to a transactional anomaly score collection. In addition, each rule will also store meta-data about each rule match (such as the Rule ID, Attack Category, Matched Location and Matched Data) for later logging. 
+For more information in anomaly scoring, click [here](https://www.modsecurity.org/CRS/Documentation/anomaly.html#anomaly-scoring-mode)
+
+#### Ingress Annotations
```

Comment:
I would move this and the logging section higher up. These are the practical details that the user needs, so I'd recommend putting them earlier in the document.

Reply:
I have moved logging and and how to section higher up in the doc